src/main/java/cs2103/v15_1j/jolt/command/ChangeCommand.java

```diff
@@ -103,13 +105,25 @@ public UIFeedback execute(ControllerStates conStates) {
             if (newStartDate != null) {
             	if (actual instanceof DeadlineTask) {
             		deadlineTask.setDate(newStartDate);
+            	} else if (actual instanceof FloatingTask) {
+            		conStates.masterList.remove(actual);
+            		if (newStartTime != null) {
```

Comment:
later in the `if (newStartTime != null)` condition, time will be set if necessary. Hence heer you don't need this condition, just set the date only, and the time being `LocalTime.MAX`